src/panels/lovelace/editor/config-elements/hui-tile-card-features-editor.ts

```diff
@@ -90,6 +93,7 @@ const SUPPORTS_FEATURE_TYPES: Record<
   "vacuum-commands": supportsVacuumCommandTileFeature,
   "water-heater-operation-modes": supportsWaterHeaterOperationModesTileFeature,
   "select-options": supportsSelectOptionTileFeature,
+  number: supportsNumberTileFeature,
```

Comment:
```suggestion
  "number": supportsNumberTileFeature,
```
this should be added for consistency

Reply:
Formatting is automatically done by prettier.